README.md

```diff
@@ -140,35 +140,57 @@ To register a [resource controller](https://laravel.com/docs/controllers#resourc
 
 You can use `only` or `except` parameters to manage your resource routes availability.
 
+You can use `parameters` parameter to modify the default parameters set by the resource attribute.
+
 You can use the `names` parameter to set the route names for the resource controller actions. Pass a string value to set a base route name for each controller action or pass an array value to define the route name for each controller action.
 
+You can use `shallow` parameter to make a nested resource to apply nesting only to routes without a unique child identifier (`index`, `create`, `store`).
+
+You can use `apiResource` parameter to only include actions used in APIs. Alternatively, you can use `ApiResource` attribute.
```

Comment:
It's not clear to me what the `ApiResource` actually does. Could you add an example of that?

Reply:
I added some more explanation about the ApiResource and a commented alternative approach in the example. How does that look?